builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -538,15 +538,18 @@ func resourceAwsS3BucketRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
-	//read the acceleration status
-	accelerate, err := s3conn.GetBucketAccelerateConfiguration(&s3.GetBucketAccelerateConfigurationInput{
-		Bucket: aws.String(d.Id()),
-	})
-	log.Printf("[DEBUG] S3 bucket: %s, read Acceleration: %v", d.Id(), accelerate)
-	if err != nil {
-		return err
+	s3HostedRegion := meta.(*AWSClient).region
+	if s3HostedRegion != "cn-north-1" && s3HostedRegion != "us-gov-west-1" {
```

Comment:
@radeksimko yes, we have, and with it it would be not necessarily better, as per:

```
isAWSErr(err, "UnsupportedArgument", "The request contained an unsupported argument")
```

I mean, this is fine too, although very verbose. In this particular case accessing the "ArgumentName" would be a plus, but sadly this is not exposed via the error interface.